stdlib/kokorawr/kokorawr/functions/services/scores.js

```diff
@@ -1,39 +1,41 @@
 const lib = require('lib')({token: process.env.STDLIB_TOKEN});
 
+const winInc = {'bs': 31, 'c4': 23, 'ddg': [1, -5], 'mm': 37, 'rps': 3, 'ttt': 11}
+
 /**
 * Leaderboard information for teams!!!
 * @param {boolean} post if you're trying to store information
-* @param {object} store information to store. Format should be {name: str, info: gameInfo}
+* @param {integer} team which team should increase their score
+* @param {string} game which game was just played
+* param {integer} index which score to use
 * @returns {object} {<game_name>: [red_score, blue_score]}
 */
-module.exports = (post=false, store={}, context, callback) => {
-  lib.utils.storage.get('scores', (err, scores) => {
+module.exports = (post=false, team, game, index=3, context, callback) => {
```

Comment:
You should put the optional parameters after the required parameters. Mixing is bad.

Reply:
made it consistent win the other files 🌮